pkg/box/box.go

```diff
@@ -56,6 +56,8 @@ type OktaConfig struct {
 	OIDCClientID string `yaml:"oidcClientID"`
 	// OrgDomain is the hostname of the Okta instance.
 	OrgDomain string `yaml:"orgDomain"`
+	// SessionDuration is the TTL for the session.
+	SessionDuration string `yaml:"sessionDuration"`
```

Comment:
suggestion: we made this a number in box (which may not have been the best choice on my part), which means it needs to be a number here.
```suggestion
	SessionDuration uint `yaml:"sessionDuration"`
```